src/Generator.php

```diff
@@ -131,7 +131,7 @@ public function generate($name, $data = [])
      * @throws Exception\RouteNotFound
      *
      */
-    public function generateRaw($name, $data = [])
+    public function generateRaw($name, array $data = [])
```

Comment:
could you please add `array` to build method also ?

https://github.com/auraphp/Aura.Router/pull/132/files#diff-7961978656794739d73aa8a22eb6b8bcR153